testing/openshift-junit/src/main/java/io/dekorate/testing/openshift/OpenshiftExtension.java

```diff
@@ -185,36 +188,6 @@ public void afterAll(ExtensionContext context) {
      } catch (Exception e) {}
   }
 
-
-  public void build(ExtensionContext context, Project project) {
```

Comment:
Is this removal related somehow to the diagnostics?

Reply:
No, its dead code elimination.

Reply:
OK, a separate commit perhasp?

Reply:
removed that completly from this pr

Reply:
OK :)